optuna/integration/tensorflow.py

```diff
@@ -21,8 +21,8 @@ class TensorFlowPruningHook(SessionRunHook):
 
     Example:
 
-        See `the example <https://github.com/optuna/optuna/blob/30dba8814d332ff4
-        0d9c7465ee660a8ecf499547/examples/tensorflow_estimator_simple.py>`_
```

Comment:
Seems like wrong example. Should it not be https://github.com/optuna/optuna/blob/master/examples/pruning/tensorflow_estimator_integration.py ? Might be beyond the scope of this PR but I think we might as well fix it here as it's highly related.

Reply:
Nice catch! Thanks.
I fixed to point to correct link.